packages/ember-htmlbars/tests/helpers/yield_test.js

```diff
@@ -48,266 +42,6 @@ QUnit.module('ember-htmlbars: Support for {{yield}} helper', {
-
-  equal(view.$('div p:contains(update) + p:contains(update)').length, 1, 'keyword has correctly propagated update');
-});
-
-QUnit.test('yield view should be a virtual view', function() {
-  var component = Component.extend({
-    isParentComponent: true,
-
-    layout: compile('{{yield}}')
-  });
-
-  view = EmberView.create({
-    template: compile('{{#view component}}{{view includedComponent}}{{/view}}'),
-    controller: {
-      component: component,
-      includedComponent: Component.extend({
-        didInsertElement() {
-          var parentView = this.get('parentView');
-
-          ok(parentView.get('isParentComponent'), 'parent view is the parent component');
-        }
-      })
-    }
-  });
-
-  runAppend(view);
-});
-
-
-QUnit.test('yield should work for views even if parentView is null', function() {
-  view = EmberView.create({
-    layout:   compile('Layout: {{yield}}'),
-    template: compile('View Content')
-  });
-
-  run(function() {
-    view.createElement();
-  });
-
-  equal(view.$().text(), 'Layout: View Content');
-});
-
-QUnit.test('simple bindings inside of a yielded template should work properly when the yield is nested inside of another view', function() {
-  view = EmberView.create({
-    layout:   compile('{{#if view.falsy}}{{else}}{{yield}}{{/if}}'),
-    template: compile('{{view.text}}'),
-    text: 'ohai'
-  });
-
-  run(function() {
-    view.createElement();
-  });
-
-  equal(view.$().text(), 'ohai');
-});
-
-QUnit.test('nested simple bindings inside of a yielded template should work properly when the yield is nested inside of another view', function() {
-  view = EmberView.create({
-    layout:   compile('{{#if view.falsy}}{{else}}{{yield}}{{/if}}'),
-    template: compile('{{#if view.falsy}}{{else}}{{view.text}}{{/if}}'),
-    text: 'ohai'
-  });
-
-  run(function() {
-    view.createElement();
-  });
-
-  equal(view.$().text(), 'ohai');
-});
-
-QUnit.module('ember-htmlbars: Component {{yield}}', {
-  setup() {
-    commonSetup();
-    originalViewKeyword = registerKeyword('view',  viewKeyword);
-  },
-  teardown() {
-    runDestroy(view);
-    delete helpers['inner-component'];
-    delete helpers['outer-component'];
-    resetKeyword('view', originalViewKeyword);
-  }
-});
-
-QUnit.test('yield with nested components (#3220)', function() {
-  var InnerComponent = Component.extend({
-    layout: compile('{{yield}}')
-  });
-
-  owner.register('component:inner-component', InnerComponent);
-
-  var OuterComponent = Component.extend({
-    layout: compile('{{#inner-component}}<span>{{yield}}</span>{{/inner-component}}')
-  });
-
-  owner.register('component:outer-component', OuterComponent);
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile(
-      '{{#outer-component}}Hello world{{/outer-component}}'
-    )
-  }).create();
-
-  runAppend(view);
-
-  equal(view.$('div > span').text(), 'Hello world');
-});
-
 QUnit.test('view keyword works inside component yield', function () {
```

Comment:
this test is about the view keyword in the yield. To my understanding this will not be supported in Glimmer2